lib/customer.rb

```diff
@@ -0,0 +1,53 @@
+# This is the main class of the program
+# It allows a customer to look, decide, and place orders from a menu
+
+class Customer
```

Comment:
I find the name of this class confusing, because as a user of the takeaway application I wouldn't expect to tell a 'customer' to do things. It looks like this class is an interface to other functionality, it's worth asking 'what is the purpose of this class?'. It might not need to exist, or it might need to be renamed to make it's purpose clearer.